packages/core/src/util/log.ts

```diff
@@ -9,6 +9,10 @@ export function log(message?: any, ...optionalParams: any[]) {
 }
 
 export function error(message?: any, ...optionalParams: any[]) {
+  // e.g. when there is a jest test like instrumentHtml.test.ts
+  // that intentionally throws an error, don't log from a
+  // try-catch that logs and re-raises the error
+  if ("test" === NODE_ENV) return;
```

Comment:
another way to achieve is a spy on console.error. If we wanted this globally, could do it in the setup.